chart/etcd-backup-restore/templates/etcd-configmap.yaml

```diff
@@ -55,15 +71,17 @@ data:
 
 {{- if .Values.etcdTLS }}
```

Comment:
Please also add peer TLS information configs here, based on whether peer TLS is enabled or not in `values.yaml`.